### PR TITLE
Give option to exclude other packages.

### PR DESCRIPTION
**Pull request by floitsch**
Our viewer can't handle them anyway, and they take up a lot of space.

### CODE REVIEW

File: tools/toitlsp/lsp/toitdoc/toitdoc.go [hunk 1]
```diff
@@ -184,10 +187,17 @@ func (b *builder) build() *Doc {
 		Libraries:  Libraries{},
 	}
 
+	packageURI := uri.PathToURI(b.rootPath) + "/.packages/"
 	for u, m := range b.summaries {
 		if b.excludeSDK && strings.HasPrefix(string(u), string(b.sdkURI)) {
 			continue
 		}
+		// TODO(florian): this is a complete hack.
```
Comment (kasperl) [inline tools/toitlsp/lsp/toitdoc/toitdoc.go:195]: Maybe elaborate a tiny bit on what makes it a hack?

File: tools/toitlsp/lsp/toitdoc/toitdoc.go [hunk 2]
```diff
@@ -184,10 +187,17 @@ func (b *builder) build() *Doc {
...
+		// However, we don't have easy access to where the project was located, so it's
+		// not trivial to
```
Comment (kasperl) [inline tools/toitlsp/lsp/toitdoc/toitdoc.go:197]: Complete sentence?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tools/toitlsp/lsp/toitdoc/toitdoc.go (hunk 1 above)
Comment (floitsch) [reply tools/toitlsp/lsp/toitdoc/toitdoc.go]: Initially it was `Contains(stringu(u), "/.packages/")`. I then improved on it, but forgot to remove the TODO.

File: tools/toitlsp/lsp/toitdoc/toitdoc.go (hunk 2 above)
Comment (floitsch) [reply tools/toitlsp/lsp/toitdoc/toitdoc.go]: Moved and updated.
done.

**Pull request merged**